app/sentinel/cmd/parse.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Note to self: these copyright banners will need to change “again”.

(will wait for Broadcom’s update about it.